criteria/geode/src/org/immutables/criteria/geode/GeodeQueryVisitor.java

```diff
@@ -165,6 +220,13 @@ private Oql oql(String oql) {
     return new Oql(variables, oql);
   }
 
+  private static String toMethod(Operator op) {
```

Comment:
What is the value of this method if it handles just one operator ? 
Maybe remove or add `size` ?

Reply:
fair point but I will be adding other operators for String (`matches, contains, etc`)